docs/data/sensitive-data/sd-desktop-secondary-working.md

```diff
@@ -0,0 +1,145 @@
+
+- [Introduction to Linux](sd-desktop-secondary-working.md#introduction-to-linux)
+- [Security-related features and limitations](sd-desktop-secondary-working.md#security-related-features-and-limitations)
+- [Accessing the secure environment](sd-desktop-secondary-working.md#accessing-the-secure-environment)
+- [Copy-paste from your laptop to virtual desktop](sd-desktop-secondary-working.md#copy-paste-from-your-laptop-to-virtual-desktop)
+- [Default software available in SD Desktop](sd-desktop-secondary-working.md#default-software-available-in-sd-desktop)
+- [Accessing RStudio](sd-desktop-secondary-working.md#accessing-rstudio)
+
+## Introduction to Linux
+
+Linux is an open-source operating system, meaning it’s free to use, and its code is openly available for people to view and modify.
+It's commonly used in many research applications due to its stability, security, and flexibility.
+
+- **Look and feel**: Linux may look slightly different from Windows, depending on the “desktop environment” used. But don’t worry – it still has familiar elements like windows folders and menus. You will find the main menu at the top left corner, where you can access all available software.
+  
+- **File structure**: Instead of “My Documents” or drives labeled C: or D:, Linux has a different way of organizing files. The main directory starts with /, and you’ll see folders like /home for your personal files.
+  
+- **Benefits**: Linux is very stable and less likely to crash, making it ideal for long or intensive computing tasks.
+
+## Security-related features and limitations
+
+SD Desktop is a secure environment designed specifically for analysing sensitive data and the services has certain features that may work differently than a regular computer. For each of these features, a specific step-by-step guide is available below. If you are using the service for the first time, reviewing these guides will help you become familiar with how everything works.
+
+- **Isolated from the internet**: your virtual desktop is completely isolated from the internet. This means that although you can open a web browser like Firefox, you will not be able to access websites or online repositories directly. This feature helps keep your data safe by reducing the risk of online threats.
+  
+- **Encrypted file only**: Only encrypted files are accessible from your virtual desktop. All files exported from the virtual desktop also need to be encrypted. This adds an extra layer of protection for your data.
+  
+- **Controlled file access with Data Gateway**: The import of data and software is restricted in SD Desktop. You cannot import any data or software yourself for security reasons. If you are working with a dataset for which you have received a permit from a data controller, the only way to access the data for analysis is by utilizing a specific application called [Data Gateway](./sd-desktop-secondary-access.md) and imported only via the SD Apply service (upon approval for reuse).
```

Comment:
Maybe this could be left out, as Findata doesn't approve access in SD Apply, it is automatic.